pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.609.1</version>
```

Comment:
Is this now requiring a much newer version of Jenkins? I don't think that's what we want unless there's a strong reason.